spreadsheet/formula/invalidreferencecontext.go

```diff
@@ -30,3 +30,27 @@ func (i *ivr) Sheet(name string) Context {
 func (i *ivr) SetOffset(col, row uint32) {
 
 }
+
+func (i *ivr) GetFormat(cellRef string) string {
```

Comment:
godoc, are these supposed to be empty?

Reply:
Yes, they are supposed to be empty and present there only to implement the interface. This type implements interface Context of formula package and all its methods are documented in the definition of the interface (spreadsheet/formula/context.go). Do we really need to godoc it?

Reply:
If it's more specific.  For example here.
// GetFormat returns an empty string for the invalid reference context.
It's better to keep the style consistent.
If golint complains about it, we should do it (most often at least).

Reply:
done